wrapper/src/vmaf.cpp

```diff
@@ -823,83 +827,117 @@ std::unique_ptr<LibsvmNusvrTrainTestModel> BootstrapVmafQualityRunner::_load_mod
-    double scoreMinusDelta = predictionMap[VmafPredictionReturnType::MINUS_DELTA];
-    double& scoreStdDev = predictionMap[VmafPredictionReturnType::STDDEV];
+    double scorePlusDelta = predictionStruct.vmafPrediction[VmafPredictionReturnType::PLUS_DELTA];
+    double scoreMinusDelta = predictionStruct.vmafPrediction[VmafPredictionReturnType::MINUS_DELTA];
+    double& scoreStdDev = predictionStruct.vmafPrediction[VmafPredictionReturnType::STDDEV];
 
     double slope = (scorePlusDelta - scoreMinusDelta) / (2.0 * BootstrapVmafQualityRunner::DELTA);
     scoreStdDev *= slope;
 }
 
 void BootstrapVmafQualityRunner::_set_prediction_result(
-        std::vector<std::map<VmafPredictionReturnType, double> > predictionMaps,
+        std::vector<VmafPredictionStruct> predictionStructs,
         Result& result) {
 
-    VmafQualityRunner::_set_prediction_result(predictionMaps, result);
+    VmafQualityRunner::_set_prediction_result(predictionStructs, result);
 
     StatVector baggingScore, stdDev, ci95LowScore, ci95HighScore;
-    for (size_t i = 0; i < predictionMaps.size(); i++) {
-        baggingScore.append(predictionMaps.at(i)[VmafPredictionReturnType::BAGGING_SCORE]);
-        stdDev.append(predictionMaps.at(i)[VmafPredictionReturnType::STDDEV]);
-        ci95LowScore.append(predictionMaps.at(i)[VmafPredictionReturnType::CI95_LOW]);
-        ci95HighScore.append(predictionMaps.at(i)[VmafPredictionReturnType::CI95_HIGH]);
+    for (size_t i = 0; i < predictionStructs.size(); i++) {
+        baggingScore.append(predictionStructs.at(i).vmafPrediction[VmafPredictionReturnType::BAGGING_SCORE]);
+        stdDev.append(predictionStructs.at(i).vmafPrediction[VmafPredictionReturnType::STDDEV]);
+        ci95LowScore.append(predictionStructs.at(i).vmafPrediction[VmafPredictionReturnType::CI95_LOW]);
+        ci95HighScore.append(predictionStructs.at(i).vmafPrediction[VmafPredictionReturnType::CI95_HIGH]);
     }
     result.set_scores("bagging", baggingScore);
     result.set_scores("stddev", stdDev);
     result.set_scores("ci95_low", ci95LowScore);
     result.set_scores("ci95_high", ci95HighScore);
+
+    // num_models is same across frames, so just use first frame length
+    int num_models = predictionStructs.at(0).vmafMultiModelPrediction.size();
+    std::vector<double> perModelScore;
+    // character array to put the name of the vmaf bootstrap model, e.g. vmaf_0001 is the first one
+    char char_buffer[50];
+
+    for (int j = 0; j < num_models; j++) {
+        for (size_t i = 0; i < predictionStructs.size(); i++) {
+            perModelScore.push_back(predictionStructs.at(i).vmafMultiModelPrediction.at(j));
+        }
+        sprintf(char_buffer, "%04d", j + 1);
+        result.set_scores("vmaf_" + std::string(char_buffer), perModelScore);
+        perModelScore.clear();
+    }
+
 }
 
-static const char VMAFOSS_DOC_VERSION[] = "1.3.7";
+static const char VMAFOSS_DOC_VERSION[] = "1.3.10";
```

Comment:
Actually it should be 1.3.11 now. Could you also update three things: 1) VERSION, 2) CHANGELOG.md, 3) wrapper/libvmaf.pc?